csharp/examples/FlightClientExample/Program.cs

```diff
@@ -0,0 +1,130 @@
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Grpc.Net.Client;
+using Apache.Arrow.Flight.Client;
+using Apache.Arrow.Flight;
+using Apache.Arrow;
+using System.Linq;
+using System;
+using System.Threading;
+using System.Runtime.CompilerServices;
+using System.Collections.Generic;
+
+namespace FlightClientExample
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            string host = args.Length > 0 ? args[0] : "localhost";
+            string port = args.Length > 1 ? args[1] : "433";
+
+            // Create client
+            // (In production systems, you should use https not http)
+            var address = $"http://{host}:{port}";
+            Console.WriteLine($"Connecting to: {address}");
+            var channel = GrpcChannel.ForAddress(address);
+            var client = new FlightClient(channel);
+
+            var recordBatches = new RecordBatch[] {
+                CreateTestBatch(0, 2000), CreateTestBatch(50, 9000)
+            };
+
+            // Particular flights are identified by a descriptor. This might be a name,
+            // a SQL query, or a path. Here, just using the name "test".
+            var descriptor = FlightDescriptor.CreateCommandDescriptor("test");
+
+            // Upload data with StartPut
+            var batchStreamingCall = client.StartPut(descriptor);
+            foreach (var batch in recordBatches) {
+                await batchStreamingCall.RequestStream.WriteAsync(batch);
+            }
+            // Signal we are done sending record batches
+            await batchStreamingCall.RequestStream.CompleteAsync();
+            // Retrieve final response
+            await batchStreamingCall.ResponseStream.MoveNext(default);
+            Console.WriteLine(batchStreamingCall.ResponseStream.Current.ApplicationMetadata.ToStringUtf8());
+            Console.WriteLine($"Wrote {recordBatches.Length} batches to server.");
+
+            // Request information:
+            var schema = await client.GetSchema(descriptor).ResponseAsync;
+            Console.WriteLine($"Schema saved as: \n {schema.ToString()}");
+
+            var info = await client.GetInfo(descriptor).ResponseAsync;
+            Console.WriteLine($"Info provided: \n {info.ToString()}");
```

Comment:
```suggestion
            Console.WriteLine($"Schema saved as: \n {schema}");

            var info = await client.GetInfo(descriptor).ResponseAsync;
            Console.WriteLine($"Info provided: \n {info}");
```

No need to call ToString here